packages/opentelemetry-core/src/trace/NoopSpan.ts

```diff
@@ -15,30 +15,18 @@
  */
 
 import * as types from '@opentelemetry/types';
-import { randomSpanId, randomTraceId } from '../common/util/id';
 
-/** This is a base/default implementation of {@link Span}. */
-export class BaseSpan implements types.Span {
-  name = 'DefaultSpan'; // default name
-  spanId = '';
-  traceId = '';
-  parentSpanId = '';
-  kind = types.SpanKind.INTERNAL; // default kind
+const INVALID_SPAN_CONTEXT = { traceId: '', spanId: '' };
 
-  constructor() {
-    // TODO: Consider to add SpanOptions with name, kind, spanContext properties
-    this.traceId = randomTraceId();
-    this.spanId = randomSpanId();
-  }
-
-  // By default returns a no-op SpanContext.
+/**
+ * The NoopSpan is the default {@link Span} that is used when no Span
+ * implementation is available. All operations are no-op except context
+ * propagation.
+ */
+export class NoopSpan implements types.Span {
```

Comment:
While I agree with the name change, why was it done? I know there were discussions in the past about this name and it was rejected.

Reply:
Just saw your answer in https://github.com/open-telemetry/opentelemetry-js/pull/45#discussion_r299744439 👍